# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/RecordedEvent.cpp
# Commit: a79d9a152a6d
# Full Hash: a79d9a152a6d52a71210d4675d3c86b94615cd1b
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-08-04 21:45:54
# Regressor Bug: 1722300
# File Overlap Count: 1
# Description:
#   Bug 1722300 - patch 4 - Implement internal destinations when generating PDF output through cairo. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120960
# ==============================================================================

diff -r b11f7f6bff04 -r a79d9a152a6d gfx/2d/RecordedEvent.cpp
--- a/gfx/2d/RecordedEvent.cpp	Wed Aug 04 12:52:37 2021 +0000
+++ b/gfx/2d/RecordedEvent.cpp	Wed Aug 04 12:52:38 2021 +0000
@@ -117,6 +117,8 @@
       return "ExternalSourceSurfaceCreation";
     case LINK:
       return "Link";
+    case DESTINATION:
+      return "Destination";
     default:
       return "Unknown";
   }